koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -106,133 +241,246 @@ cte_ocp_on_aws_tag_joined AS (
+    cast(day(usage_start) as varchar) as day
+FROM (
+    SELECT pds.aws_uuid,
+        max(pds.data_source) as data_source,
+        pds.namespace,
+        max(pds.node) as node,
+        max(pds.persistentvolumeclaim) as persistentvolumeclaim,
+        max(pds.persistentvolume) as persistentvolume,
+        max(pds.storageclass) as storageclass,
+        max(pds.resource_id) as resource_id,
+        max(pds.usage_start) as usage_start,
+        max(pds.usage_end) as usage_end,
+        max(pds.product_code) as product_code,
+        max(pds.product_family) as product_family,
+        max(pds.instance_type) as instance_type,
+        max(pds.usage_account_id) as usage_account_id,
+        max(aa.id) as account_alias_id,
+        max(pds.availability_zone) as availability_zone,
+        max(pds.region) as region,
+        max(pds.unit) as unit,
+        sum(pds.usage_amount) as usage_amount,
+        max(pds.currency_code) as currency_code,
+        sum(pds.unblended_cost) as unblended_cost,
+        sum(pds.markup_cost) as markup_cost,
+        sum(pds.pod_usage_cpu_core_hours) as pod_usage_cpu_core_hours,
+        sum(pds.pod_request_cpu_core_hours) as pod_request_cpu_core_hours,
+        sum(pds.pod_limit_cpu_core_hours) as pod_limit_cpu_core_hours,
+        sum(pds.pod_usage_memory_gigabyte_hours) as pod_usage_memory_gigabyte_hours,
+        sum(pds.pod_request_memory_gigabyte_hours) as pod_request_memory_gigabyte_hours,
+        max(pds.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
+        max(pds.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
+        max(pds.pod_labels) as pod_labels,
+        max(pds.volume_labels) as volume_labels,
+        max(pds.tags) as tags,
+        max(pds.project_rank) as project_rank,
+        max(pds.data_source_rank) as data_source_rank,
+        max(pds.resource_id_matched) as resource_id_matched
+    FROM hive.{{schema | sqlsafe}}.reporting_ocpawscostlineitem_project_daily_summary_temp AS pds
+    LEFT JOIN postgres.{{schema | sqlsafe}}.reporting_awsaccountalias AS aa
+        ON pds.usage_account_id = aa.account_id
+    GROUP BY aws_uuid, namespace
+) as ocp_aws
+;
+
+INSERT INTO postgres.{{schema | sqlsafe}}.reporting_ocpawscostlineitem_project_daily_summary (
+    uuid,
+    report_period_id,
+    cluster_id,
+    cluster_alias,
+    data_source,
+    namespace,
+    node,
+    persistentvolumeclaim,
+    persistentvolume,
+    storageclass,
+    resource_id,
+    usage_start,
+    usage_end,
+    product_code,
+    product_family,
+    instance_type,
+    cost_entry_bill_id,
+    usage_account_id,
+    account_alias_id,
+    availability_zone,
+    region,
+    unit,
+    usage_amount,
+    currency_code,
+    unblended_cost,
+    markup_cost,
+    pod_cost,
+    project_markup_cost,
+    pod_labels,
+    tags,
+    source_uuid
+)
+SELECT uuid(),
+    {{report_period_id | sqlsafe}} as report_period_id,
+    {{cluster_id}} as cluster_id,
```

Comment:
I think you may need to replace the `{{cluster_id}}` & `{{cluster_alias}}` here as well.